crates/bevy_text/src/pipeline.rs

```diff
@@ -300,7 +300,9 @@ pub struct TextLayoutInfo {
 ///
 /// Generated via [`TextPipeline::create_text_measure`].
 pub struct TextMeasureInfo {
+    /// Minimum size for a text area
```

Comment:
In pixels? Or something else? Clarifications about how this represents a rectangle would be useful too (or we can swap to a clearer type).

Reply:
Hmm, from what I can find, it seems this would be in pixels, since the only place it is used is in the textwidget under bevy_ui, which only referenses pixels...